infra/feast-operator/config/samples/v1alpha1_featurestore_postgres_db_volumes_ssl.yaml

```
@@ -0,0 +1,167 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres-secret
+  namespace: default
+  labels:
+    app: postgres
+stringData:
+  POSTGRES_DB: mydatabase
+  POSTGRES_USER: admin
+  POSTGRES_PASSWORD: password
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: feast-data-stores
+  namespace: default
+stringData:
+  redis: |
+    connection_string: redis.feast.svc.cluster.local:6379
+  sql: |
+    path: postgresql+psycopg://admin:password@postgresql.default.svc.cluster.local:5432/mydatabase?sslmode=require&sslrootcert=/var/lib/postgresql/certs/ca.crt&sslcert=/var/lib/postgresql/certs/tls.crt&sslkey=/var/lib/postgresql/certs/tls.key
+    cache_ttl_seconds: 60
+    sqlalchemy_config_kwargs:
+        echo: false
+        pool_pre_ping: true
+  postgres: |
+    host: postgresql.default.svc.cluster.local
+    port: 5432
+    database: mydatabase
+    db_schema: public
+    user: admin
+    password: password
+    sslmode: require
+    sslkey_path: /var/lib/postgresql/certs/tls.key
+    sslcert_path: /var/lib/postgresql/certs/tls.crt
+    sslrootcert_path: /var/lib/postgresql/certs/ca.crt
+    vector_enabled: false
+    vector_len: 512
+  postgres_offline: |
+    host: postgresql.default.svc.cluster.local
+    port: 5432
+    database: mydatabase
+    db_schema: public
+    user: admin
+    password: password
+    sslmode: require
+    sslkey_path: /var/lib/postgresql/certs/tls.key
+    sslcert_path: /var/lib/postgresql/certs/tls.crt
+    sslrootcert_path: /var/lib/postgresql/certs/ca.crt
+---
+apiVersion: feast.dev/v1alpha1
+kind: FeatureStore
+metadata:
+  name: example
+  namespace: default
+spec:
+  feastProject: credit_scoring_local
+  services:
+    volumes:
+      - name: ca-cert
+        secret:
+          secretName: postgresql-client-certs
+          items:
+            - key: ca.crt
+              path: ca.crt
+              mode: 0644  # Readable by all, required by PostgreSQL
+
+      - name: client-cert
+        secret:
+          secretName: postgresql-client-certs
+          items:
+            - key: tls.crt
+              path: tls.crt
+              mode: 0644  # Required for the client certificate
+
+      - name: client-key
+        secret:
+          secretName: postgresql-client-certs
+          items:
+            - key: tls.key
+              path: tls.key
+              mode: 0640  # Required for the private key
```

i wonder if we can simplify here...
```suggestion
      - name: postgres-certs
        secret:
          secretName: postgresql-client-certs
          items:
            - key: ca.crt
              path: ca.crt
              mode: 0644  # Readable by all, required by PostgreSQL
            - key: tls.crt
              path: tls.crt
              mode: 0644  # Required for the client certificate
            - key: tls.key
              path: tls.key
              mode: 0640  # Required for the private key
```